watchgod/watcher.py

```diff
@@ -99,8 +99,14 @@ def should_watch_file(self, entry: 'DirEntry') -> bool:
 
 
 class PythonWatcher(DefaultDirWatcher):
+    def __init__(self, *args, extensions: List[str]=[], **kwargs):
```

Comment:
This is going to cause problems since lists are mutable, they should not be used as default arguments.

Instead, please use a tuple or use `None` as the default argument.